### ISSUE TITLE
Overriding file name with time

### ISSUE THREAD

**Issue by FunnyPocketBook** [ihabunek/twitch-dl#89]
When overriding the file name with `{datetime}` or `{time}`, the joining of the files results in an error, as the colons are disallowed.

Command executed:
```twitch-dl download 1276935029 -s 04:57:10 -o {date}.mkv```
Output:
```
ffmpeg -i /tmp/twitch-dl/3c9c6b44a5e54d109bc9_jvckk_44509094204_1643230617/720p60/playlist_downloaded.m3u8 -c copy -metadata artist=jvckk -metadata title=MODDED AMOGUS > THE GAME OF LIFE 2 -metadata encoded_by=twitch-dl -stats -loglevel warning 2022-01-26T20:57:04Z.mkv
2022-01-26T20:57:04Z.mkv: Protocol not found
Joining files failed
```

**Comment by ihabunek** [comment]
Thanks for reporting. It's not that they're disallowed, it's just that ffmpeg gets confused because it uses colons for something else.

**Comment by ihabunek** [comment]
Fixed in [1.18.1](https://github.com/ihabunek/twitch-dl/releases/tag/1.18.1).
